internal/webhook/module.go

```diff
@@ -112,10 +113,15 @@ func validateModule(mod *kmmv1beta1.Module) (admission.Warnings, error) {
 }
 
 func validateModuleLoaderContainerSpec(container kmmv1beta1.ModuleLoaderContainerSpec) error {
+
 	if container.InTreeModulesToRemove != nil && container.InTreeModuleToRemove != "" { //nolint:staticcheck
 		return fmt.Errorf("only one of the Container's fields: InTreeModulesToRemove or InTreeModuleToRemove can be defined")
 	}
 
+	if contImg := container.ContainerImage; contImg != "" && !strings.Contains(contImg, ":") && !strings.Contains(contImg, "@") {
```

Comment:
WDYT about using NewTag and NewDigest from ithub.com/google/go-containerregistry? It provides a little bit more verifications on the input

Reply:
Fixed.